scripts/set_mem_constraints.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+# Abort script on any error
+set -e
+
+#####################################################################
+## Portions of this script is copied from:
+## https://github.com/fabric8io-images/java/blob/master/images/jboss/openjdk8/jdk/run-java.sh
```

Comment:
replace this with a permalink incase they change directories around again https://github.com/fabric8io-images/java/blob/14e31db422828a08ae25ff94ce3966b447ea28a8/images/jboss/openjdk8/jdk/run-java.sh